src/api/config.rs

```diff
@@ -300,13 +307,13 @@ impl SpeedSettings {
   /// - 9: min block size 64x64, reduced TX set, TX domain distortion, fast deblock.
   /// - 8: min block size 8x8, reduced TX set, TX domain distortion, fast deblock.
   /// - 7: min block size 8x8, reduced TX set, TX domain distortion.
-  /// - 6: min block size 8x8, reduced TX set, TX domain distortion.
-  /// - 5 (default): min block size 8x8, reduced TX set, TX domain distortion, complex pred modes for keyframes.
-  /// - 4: min block size 8x8, TX domain distortion, complex pred modes for keyframes.
-  /// - 3: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision.
-  /// - 2: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision, include near MVs, quantizer RDO.
-  /// - 1: min block size 8x8, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO.
-  /// - 0 (slowest): min block size 4x4, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO, bottom-up encoding.
+  /// - 6 (default): min block size 8x8, reduced TX set, TX domain distortion, complex pred modes for keyframes.
+  /// - 5: min block size 8x8, TX domain distortion, complex pred modes for keyframes.
+  /// - 4: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision.
+  /// - 3: min block size 8x8, TX domain distortion, complex pred modes for keyframes, RDO TX decision, include near MVs, quantizer RDO.
+  /// - 2: min block size 8x8, TX domain distortion, complex pred modes, RDO TX decision, include near MVs, quantizer RDO.
```

Comment:
Looks like this should say 4x4.

Reply:
I wonder if it makes sense to keep this text duplicated. We have the same text here and in the binary help output.

Reply:
@lu-zero, Sure-, lets ask other (including luca)'s opinion.
Also, "TX domain distortion" is wrong since it is not used by default as filed in https://github.com/xiph/rav1e/issues/1689 due to Psychovisual default.

Reply:
> Looks like this should say 4x4.

THANKS!

Reply:
If we would like to keep it in sync w/out duplication there are ways.

Reply:
> If we would like to keep it in sync w/out duplication there are ways.

Would you show us the ways, dude?

Reply:
Sure :) Patches coming on top of https://github.com/xiph/rav1e/pull/1673

Reply:
Hey- thanks a lot!

Reply:
You get all the fields:
```
preset 0: min_block_size = BLOCK_4X4, multiref = true, fast_deblock = false, reduced_tx_set = false, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = true, rdo_tx_decision = true, prediction_modes = ComplexAll, include_near_mvs = true, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = true, use_satd_subpel = true
preset 1: min_block_size = BLOCK_4X4, multiref = true, fast_deblock = false, reduced_tx_set = false, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true, prediction_modes = ComplexAll, include_near_mvs = true, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = true, use_satd_subpel = true
preset 2: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true, prediction_modes = ComplexKeyframes, include_near_mvs = true, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = true, use_satd_subpel = true
preset 3: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true, prediction_modes = ComplexKeyframes, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 4: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = ComplexKeyframes, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 5: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = ComplexKeyframes, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 6: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = Simple, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 7: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = Simple, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 8: min_block_size = BLOCK_8X8, multiref = false, fast_deblock = true, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = Simple, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 9: min_block_size = BLOCK_64X64, multiref = false, fast_deblock = true, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = Simple, include_near_mvs = false, no_scene_detection = false, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
preset 10: min_block_size = BLOCK_64X64, multiref = false, fast_deblock = true, reduced_tx_set = true, tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false, prediction_modes = Simple, include_near_mvs = false, no_scene_detection = true, diamond_me = true, cdef = true, quantizer_rdo = false, use_satd_subpel = false
```

Reply:
```
    -s, --speed <SPEED>
            Speed level (0 is best quality, 10 is fastest)
            Speeds 10 and 0 are extremes and are generally not recommended
            -0:
            min_block_size = BLOCK_4X4, multiref = true, fast_deblock = false, reduced_tx_set = false,
            tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = true, rdo_tx_decision = true,
            prediction_modes = ComplexAll, include_near_mvs = true, no_scene_detection = false, diamond_me = true, cdef
            = true, quantizer_rdo = true, use_satd_subpel = true
            -1:
            min_block_size = BLOCK_4X4, multiref = true, fast_deblock = false, reduced_tx_set = false,
            tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true,
            prediction_modes = ComplexAll, include_near_mvs = true, no_scene_detection = false, diamond_me = true, cdef
            = true, quantizer_rdo = true, use_satd_subpel = true
            -2:
            min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false,
            tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true,
            prediction_modes = ComplexKeyframes, include_near_mvs = true, no_scene_detection = false, diamond_me = true,
            cdef = true, quantizer_rdo = true, use_satd_subpel = true
            -3:
            min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false,
            tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = true,
            prediction_modes = ComplexKeyframes, include_near_mvs = false, no_scene_detection = false, diamond_me =
            true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
            -4:
            min_block_size = BLOCK_8X8, multiref = false, fast_deblock = false, reduced_tx_set = false,
            tx_domain_distortion = true, tx_domain_rate = false, encode_bottomup = false, rdo_tx_decision = false,
            prediction_modes = ComplexKeyframes, include_near_mvs = false, no_scene_detection = false, diamond_me =
            true, cdef = true, quantizer_rdo = false, use_satd_subpel = true
```